src/modules/map/components/cesium/CesiumMap.vue

```diff
@@ -114,6 +122,9 @@ export default {
         visiblePrimitiveLayers() {
             return this.visibleLayers.filter((l) => l instanceof GeoAdminGeoJsonLayer)
         },
+        visibleKMLLayers() {
+            return this.visibleLayers.filter((l) => l instanceof KMLLayer)
```

Comment:
I added addToMap to the condition 

About zIndex, as we discussed in slack, zIndex does not work for primitives. I think we can make it work for clamped to the ground lines and polygons but drawing always one layer and I am not sure that it will be useful